docs/guide/controller-plugin.md

```diff
@@ -68,12 +68,98 @@ Robot windows are implemented in HTML and provide the following features:
 The equivalent controller functions are `wb_robot_wwi_receive_text` and `wb_robot_wwi_send_text`.
 3. They are web-ready and could be used to display robot windows on web pages.
 
-A simple example of an HTML robot window is provided in the `robots/thymio/thymio2.wbt` sample simulation and demonstrates:
+A simple example of an HTML robot window is provided in the `projects/samples/howto/plugins/robot_windows/custom_html_robot_window/` sample simulation and demonstrates:
```

Comment:
It would be nice to transform 'projects/samples/howto/plugins/robot_windows/custom_html_robot_window' into a link to the file on github.